hal/audio_hw.c

```diff
@@ -18,7 +18,7 @@
  */
 
 #define LOG_TAG "audio_hw_primary"
-//#define LOG_NDEBUG 0
+#define LOG_NDEBUG 0
```

Comment:
You left the debug flag enabled. This should not be in in the commit.

Reply:
@mariogrip can you re-comment this line or should I ? can we merge then?